session9/src/homework/ex1.java

```diff
@@ -0,0 +1,10 @@
+package homework;
+
+import java.time.LocalDate;
+
+public class ex1 {
```

Comment:
Remember that the class names should start with uppercase letters. Please rename all of them in this package (ex1, ex2 etc)